keras_nlp/tokenizers/byte_pair_tokenizer.py

```diff
@@ -536,3 +537,57 @@ def _bpe_merge_and_update_cache(self, tokens):
             tokenized_words, axis=1, separator=" "
         )
         self.cache.insert(tokens, tokenized_words)
+
```

Comment:
each of these classes should probably have a preset property defined now, that is empty. E.g.

https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/models/backbone.py#L33-L35